rules/windows/defense_evasion_network_connection_from_windows_binary.toml

```diff
@@ -17,10 +17,64 @@ index = ["logs-endpoint.events.*", "winlogbeat-*", "logs-windows.*"]
 language = "eql"
 license = "Elastic License v2"
 name = "Unusual Network Activity from a Windows System Binary"
+note = """## Triage and analysis
+
+### Investigating Unusual Network Activity from a Windows System Binary
+
+Attackers can abuse certain trusted developer utilities to proxy the execution of malicious payloads. As those utilities are usually signed, they can bypass security controls in place that could prevent or detect direct execution.
+
+This detection rule identifies network connections done by these utilities, which can indicate abuse to execute payloads or the masquerading as one of these processes.
+
+> **Note**:
+> This investigation guide uses the [Osquery Markdown Plugin](https://www.elastic.co/guide/en/security/master/invest-guide-run-osquery.html) introduced in Elastic Stack version 8.5.0. Older Elastic Stack versions will see unrendered Markdown in this guide.
+
+#### Possible investigation steps
+
+- Investigate the process execution chain (parent process tree) for unknown processes. Examine their executable files for prevalence, whether they are located in expected locations, and if they are signed with valid digital signatures.
+- Investigate abnormal behaviors observed by the subject process, such as registry or file modifications, and any spawned child processes.
+- Investigate other alerts associated with the user/host during the past 48 hours.
+- Examine the host for derived artifacts that indicates suspicious activities:
+  - Analyze the process executable using a private sandboxed analysis system.
+  - Observe and collect information about the following activities in both the sandbox and the alert subject host:
+    - Attempts to contact external domains and addresses.
+      - Use the Elastic Defend network events to determine domains and addresses contacted by the subject process by filtering by the process' `process.entity_id`.
+      - Examine the DNS cache for suspicious or anomalous entries.
```

Comment:
Same suggestion that made [here](https://github.com/elastic/detection-rules/pull/2521/files#diff-783799e1c367669274ab7c6261c7331663ee4d7cfcb12f3f297879303e1923baR43).